### PR TITLE
[github] Add links for discord or SO in Create Issue

### PR DESCRIPTION
**Pull request by kunaltyagi**

### CODE REVIEW

**Comment by SergioRAgostinho** [comment]
The distinction between both is still a little ambiguous. My opinion on the division

Discord: brief one line questions with one line answers; looking for opinions or general feedback
SO: problems which require setting up a proper context and explanation

**Comment by kunaltyagi** [comment]
How would you word it? 😅

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: true
+
+contact_links:
+- name: Get help on Discord
+  url: https://discord.gg/JFFMAXS
+  about: Have a quick question? Not sure if your issue affects everyone reproducibly?
+- name: Get help on StackOverflow
+  url: https://stackoverflow.com/tags/point-cloud-library/
+  about: Have a programming question? Not sure if it's a issue with your code or the PCL library?
```
Comment (kunaltyagi) [inline .github/ISSUE_TEMPLATE/config.yml:9]:
```suggestion
  about: Need some support to use PCL in your project?
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: true
+
+contact_links:
+- name: Get help on Discord
+  url: https://discord.gg/JFFMAXS
+  about: Have a quick question? Not sure if your issue affects everyone reproducibly?
```
Comment (kunaltyagi) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
  about: Have a quick question about PCL or how to use PCL in your project?
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: true
+
+contact_links:
```
Comment (kunaltyagi) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
contact_links:
- name: Learn using Tutorials
  url: https://pcl-tutorials.readthedocs.io/en/master/
  about: Read the tutorials to learn how to use PCL better
```

**Pull request merged**